python/units_neutron.cpp

```diff
@@ -24,6 +24,27 @@ void init_units_neutron(py::module &m) {
       .def(py::self - py::self)
       .def(py::self * py::self)
       .def(py::self / py::self)
+      .def("__pow__",
+           [](const units::Unit &self, int power) -> units::Unit {
+             switch (power) {
+             case 0:
+               return units::dimensionless;
+             case 1:
+               return self;
+             case 2:
+               return self * self;
+             case 3:
+               return self * self * self;
+             case -1:
+               return units::Unit() / (self);
+             case -2:
+               return units::Unit() / (self * self);
+             case -3:
+               return units::Unit() / (self * self * self);
+             default:
+               throw std::runtime_error("Unsupported power of unit.");
```

Comment:
Does this also then imply that any ** operation on a variable in which the unit's were defined, would also fail if `power` >= 4 ? from what I can see `variable` does not support `__pow__` so maybe this is only relevant for making units at the point of setting them on a variable.

Reply:
- Yes, `Variable.__pow__` would need to be added eventually.
- Yes it would fail for `Variable`, unless the variable is dimensionless. This is the desired behavior.